_includes/pbc/all/install-features/202304.0/install-the-customer-access-feature.md

```diff
@@ -0,0 +1,437 @@
+
+**src/Pyz/Client/Permission/PermissionDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Client\Permission;
+
+use Spryker\Client\CustomerAccessPermission\Plugin\CustomerAccessPermissionStoragePlugin;
+use Spryker\Client\CustomerAccessPermission\Plugin\SeeAddToCartPermissionPlugin;
+use Spryker\Client\CustomerAccessPermission\Plugin\SeeOrderPlaceSubmitPermissionPlugin;
+use Spryker\Client\CustomerAccessPermission\Plugin\SeePricePermissionPlugin;
+use Spryker\Client\CustomerAccessPermission\Plugin\SeeShoppingListPermissionPlugin;
+use Spryker\Client\CustomerAccessPermission\Plugin\SeeWishlistPermissionPlugin;
+use Spryker\Client\Permission\PermissionDependencyProvider as SprykerPermissionDependencyProvider;
+
+class PermissionDependencyProvider extends SprykerPermissionDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Client\PermissionExtension\Dependency\Plugin\PermissionStoragePluginInterface>
+     */
+    protected function getPermissionStoragePlugins(): array
+    {
+        return [
+            new CustomerAccessPermissionStoragePlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Shared\PermissionExtension\Dependency\Plugin\PermissionPluginInterface>
+     */
+    protected function getPermissionPlugins(): array
+    {
+        return [
+            new SeePricePermissionPlugin(),
+            new SeeOrderPlaceSubmitPermissionPlugin(),
+            new SeeAddToCartPermissionPlugin(),
+            new SeeWishlistPermissionPlugin(),
+            new SeeShoppingListPermissionPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that everything works fine (checks should be done for not logged-in customers:
+- `SeePricePermissionPlugin` will show or hide prices at all pages depending on configuration value.
+- `SeeOrderPlaceSubmitPermissionPlugin` will allow or disallow order submitting after going through the checkout process depending on configuration value.
+- `SeeAddToCartPermissionPlugin` is responsible for "Add to Cart" button on PDP. It will be available or not depending on configuration value.
+- `SeeWishlistPermissionPlugin` takes care about "Add to Wishlist" button on PDP. It will be shown or not depending on configuration value.
+- `SeeShoppingListPermissionPlugin` will allow or disallow adding product to shopping list from PDP depending on configuration value.
+- `CustomerAccessPermissionStoragePlugin` is responsible for customer permissions retrieving.
+
+{% endinfo_block %}
+
+## Install feature frontend
+
+### Prerequisites
+
+Overview and install the necessary features before beginning the integration step.
+
+| NAME         | VERSION          |
+|--------------|------------------|
+| Spryker Core | {{page.version}} |
+
+### 1) Add translations
+
+Append glossary according to your configuration:
+
+**data/import/common/common/glossary.csv**
```

Comment:
please revert this line to be consistent with other IGs

Reply:
But the `src/data/import/glossary.csv` path is not correct, also we have a proper file path in some IGs already
- https://docs.spryker.com/docs/pbc/all/product-information-management/202204.0/install-and-upgrade/install-features/install-the-catalog-order-management-feature.html#add-translations
- https://docs.spryker.com/docs/scos/dev/feature-integration-guides/202108.0/persistent-cart-sharing-feature-integration.html#add-translations
- https://docs.spryker.com/docs/scos/dev/feature-integration-guides/202212.0/multiple-abstract-products-as-promotional-products-and-discounts-feature-integration.html#add-translations